### ISSUE TITLE
NullReferenceException when trying to add contact to Business

### ISSUE THREAD

**Issue by bwitting** [SparkDevNetwork/Rock#892]
On the Finances > Business screen, I get the following error if I try to add a person as a Business contact:

> An error occurred on the Rock RMS site on page: 
https://rock.newpointe.org:443/page/283?businessId=37637

>HttpUnhandledException in System.Web
>Message
>Exception of type 'System.Web.HttpUnhandledException' was thrown.

>Stack Trace
at System.Web.UI.Page.HandleError(Exception e)
at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
at System.Web.UI.Page.ProcessRequest(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
at System.Web.UI.Page.ProcessRequest(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
at System.Web.UI.Page.ProcessRequest()
at System.Web.UI.Page.ProcessRequest(HttpContext context)
at ASP.themes_rock_layouts_fullwidth_aspx.ProcessRequest(HttpContext context) in c:\Windows\Microsoft.NET\Framework64\v4.0.30319\Temporary ASP.NET Files\root\e22c2559\92c7e946\App_Web_qn2fd5h4.3.cs:line 0
at System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
at System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
at System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)

>-NullReferenceException in App_Web_uso2we23
Message
Object reference not set to an instance of an object.

>Stack Trace
at RockWeb.Blocks.Finance.BusinessDetail.mdAddContact_SaveClick(Object sender, EventArgs e) in c:\inetpub\wwwroot\Blocks\Finance\BusinessDetail.ascx.cs:line 471
at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)

**Comment by azturner** [comment]
@bwitting , is it possible this business may have been imported and not saved using Rock UI?  If so, try just saving business first, and then adding contact.

**Comment by bwitting** [comment]
Yes, we did import businesses with Excavator. I opened a business, clicked edit, and got this exception:

> 'ddlRecordStatus' has a SelectedValue which is invalid because it does not exist in the list of items.
 Parameter name: value

However, I created a new Business from scratch - I was able to add a contact and edit the business.  So it must be a problem with the way the businesses got imported.  I'll try to take a look at our Businesses on a database level.
